python/tvm/relay/build_module.py

```diff
@@ -213,7 +218,7 @@ def _build_module_no_factory(mod, target=None, target_host=None, params=None, mo
     return build(mod, target, params=params, mod_name=mod_name).module
 
 
-def build(ir_mod, target=None, target_host=None, params=None, mod_name="default"):
+def build(ir_mod, target=None, target_host=None, params=None, mod_name="default", executor="graph"):
```

Comment:
cc @tqchen @joresch would be great to get feedback on API changes here.

Reply:
In general, I am happy with an object being passed here, but I think that this deserves an RFC on its own. For the time being I would not overload the target further, and use this as a good excuse to write an "option object RFC"